org.eclipse.scanning.event/src/org/eclipse/scanning/event/remote/_Scannable.java

```diff
@@ -128,8 +127,7 @@ public String getUnit() {
 		if (info==null) update();  // We assume that they do not change unit.
 		return info.getUnit();

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') A "NullPointerException" could be thrown; "info" is nullable here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2259)